lib/linguist/languages.yml

```diff
@@ -860,6 +860,16 @@ C2hs Haskell:
   codemirror_mode: haskell
   codemirror_mime_type: text/x-haskell
   language_id: 45
+C3:
+  type: programming
+  color: "#969696"
+  extensions:
+  - ".c3"
+  - ".c3i"
+  - ".c3l"
+  tm_scope: source.c
+  ace_mode: c_cpp
```

Comment:
If this language is an evolution of C, wouldn't it make more sense to use C's ace mode, assuming C3 isn't supported?